test/suites/basic.sh

```diff
@@ -210,7 +215,10 @@ test_basic_usage() {
   lxc profile create unconfined
   lxc profile set unconfined security.privileged true
   lxc init testimage foo2 -p unconfined
-  [ "$(stat -c "%a" "${LXD_DIR}/containers/foo2")" = "700" ]
+  # FIXME: make this backend agnostic
+  if [ "${LXD_BACKEND}" = "dir" ]; then
+    [ "$(stat -c "%a" "${LXD_DIR}/containers/foo2")" = "700" ]
+  fi
```

Comment:
Oh, could be. Most of these I was just on autopilot for and didn't really think about.